test/admin/e2e/general/e2e.spec.ts

```diff
@@ -1024,6 +1024,38 @@ describe('General', () => {
     })
   })
 
+  test('should not open leave-without-saving modal if opening a new tab', async () => {
+    const { id } = await createPost()
+    await page.goto(postsUrl.edit(id))
+    const title = 'title'
+    await page.locator('#field-title').fill(title)
+    await saveDocHotkeyAndAssert(page)
```

Comment:
Creating a new post and saving a change is unnecessary overhead for this test. We can simply navigate to the `/create` page, fill out a field to enter modified state, then attempt to open a new tab.

```ts
await page.goto(postsUrl.create)
await page.locator('#field-title').fill('Test')
// proceed with rest of test
```